src/Symfony/Bundle/FrameworkBundle/Command/EventDispatcherDebugCommand.php

```diff
@@ -37,6 +37,7 @@ protected function configure()
                 new InputArgument('event', InputArgument::OPTIONAL, 'An event name'),
                 new InputOption('format', null, InputOption::VALUE_REQUIRED, 'The output format  (txt, xml, json, or md)', 'txt'),
                 new InputOption('raw', null, InputOption::VALUE_NONE, 'To output raw description'),
+                new InputOption('serviceId', null, InputOption::VALUE_REQUIRED, 'The service id of your event dispatcher', 'event_dispatcher'),
```

Comment:
Looking at [other commands](https://github.com/symfony/symfony/blob/2.7/src/Symfony/Bundle/FrameworkBundle/Command/CacheClearCommand.php#L36-L37) we usually separate words with a dash in option names: `--service-id=my.event_dispatcher`.